esp/services/ws_store/ws_storeService.cpp

```diff
@@ -117,6 +117,37 @@ IEspStore* CwsstoreEx::loadStoreProvider(const char* instanceName, const char* l
     return (IEspStore*) xproc();
 }
 
+bool CwsstoreEx::onListStores(IEspContext &context, IEspListStoresRequest &req, IEspListStoresResponse &resp)
+{
+    const char *user = context.queryUserId();
+    user = "rodrigo";
+    double version = context.getClientVersion();
+
+    const char * namefilter = req.getNameFilter();
+    const char * ownerfilter = req.getOwnerFilter();
+    const char * typefilter  = req.getTypeFilter();
+
+    IArrayOf<IEspStoreInfo> storeinfos;
+    Owned<IPropertyTree> stores = m_storeProvider->getStores(namefilter, ownerfilter, typefilter, new CSecureUser(user, nullptr));
```

Comment:
You may fill out the storeinfos inside the m_storeProvider->getStores() unless the getStores() has to return a IPropertyTree in other place.

Reply:
opted against this in order to keep esp generated deps out of the DALIKV library